swatch-product-configuration/src/main/resources/OpenShift/rosa.yaml

```diff
@@ -0,0 +1,35 @@
+---
+platform: OpenShift
+
+id: rosa
+
+variants:
+  - tag: rosa
+    roles:
+      - moa-hostedcontrolplane
+    product_names:
+      - OpenShift Dedicated
+
+defaults:
+  variant: rosa
+  sla: PREMIUM
+  usage: PRODUCTION
+
+billingWindow: MONTHLY
+
+metrics:
+  - id: redhat.com:rosa:cluster_hour
+    awsDimension: control_plane
```

Comment:
Per dev sync discussion 6/26, the **rhm_metric_id** fields found in tag profile for rosa weren't accurate and just noise.  Purposely omitted it from the metrics in this file